queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -195,6 +199,12 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
                             level='error'
                         )
 
+            if any('transition' in x for x in legal_filings):
```

Comment:
The business should already be loaded, so this would also change.

Reply:
This section is no longer needed and hence removed.